presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -488,6 +488,9 @@ public Signature resolveFunction(QualifiedName name, List<TypeSignature> paramet
         if (name.getSuffix().startsWith(MAGIC_LITERAL_FUNCTION_PREFIX)) {
             // extract type from function name
             String typeName = name.getSuffix().substring(MAGIC_LITERAL_FUNCTION_PREFIX.length());
+            // The RowType have <> that are not allowed in java variable names, so we replace to {}. We can remove this
+            // hack when the type ROW doesn't have them anymore (be ROW(field type+) instead of ROW<type+>(field+))
+            typeName = typeName.replace('{', '<').replace('}', '>');
```

Comment:
Unfortunately it's not :(
When it tries to store a variable of type `ROW<..>(...)` (and use the type as part of the name), it fails here: [InvokeInstruction.java](https://github.com/prestodb/presto/blob/c7bf63a2138c2e61f0edb307cefbf6b35fba25c2/presto-bytecode/src/main/java/com/facebook/presto/bytecode/instruction/InvokeInstruction.java#L431)

If I remove that check, we get another error when trying to execute the bytecode.

And if we simply strip, it's going to be complicated to recover the original name.

Fortunately, https://github.com/prestodb/presto/pull/5043 will solve this in part by changing the syntax of row type, and then I can remove that. But, still anything with `.;[/<>` will fail to compile the bytecode.